CMSgov/mdct-mcr
# add frontend banner handling and related baseline setup  (MDCT-284 step 2)

## Description
This PR is the second step toward completion of the ticket below. Fetched banners are handled now. Frontend can trigger all Banner API operations. 

Related ticket: [MDCT-284](https://qmacbis.atlassian.net/browse/MDCT-284)

### How to test
1. Make sure you change your `.env` files to match the updated ones in the readme.
2. Run and navigate to /home. View banner get loaded.
3. Navigate to /admin. Delete banner and see changes reflected across app.
4. On /admin, replace the current banner (right now it doesn't take the user's defined banner, just a dummy one). See changes reflected across app.

### Changed Dependencies
No changed deps.

## Code author checklist

## Reviewer checklist (two different people)
